crates/api_common/src/utils.rs

```diff
@@ -276,31 +275,14 @@ async fn check_community_block(
   }
 }
 
-/// Throws an error if a recipient has blocked an instance.
 #[tracing::instrument(skip_all)]
-async fn check_instance_block(
```

Comment:
This was only run from the `send_local_notifs` function, which creates comment_replies, mentions, and emails, none of which need to be blocked in an InstanceBlock.